GsaGH/Components/0_Model/CreateGridLine.cs

```diff
@@ -66,15 +78,22 @@ protected override void SolveInstance(IGH_DataAccess da) {
           midPoint.Z = 0;
           Point3d endPoint = arc.EndPoint;
           endPoint.Z = 0;
-          gridLine = new GsaGridLine(arc, label);
+          gsaGridLine = new GsaGridLine(arc, label);
         } else {
           string message = "Invalid input geometry, curve needs to be a straight line or a circular arc.";
           this.AddRuntimeWarning(message);
           return;
         }
 
-        da.SetData(0, new GsaGridLineGoo(gridLine));
-      }
+        var assembly = new ModelAssembly(null, null, new List<GsaGridLine>() {
+          gsaGridLine,
+        }, null, null, null, null, rhinoUnits, Length.Zero, true, this);
+
```

Comment:
@psarras  @DominikaLos , I dont think any change is required in this file else you will endup changing unit twice. So, kindly double check. This is just component and should be created gridline in unit of Rhino. Conversion only when all information are assembled at the time of creating GSA API model

Reply:
without that, values will be still incorrect.

Reply:
true, we don't need this